packages/twenty-server/src/modules/workflow/workflow-executor/interfaces/workflow-action.interface.ts

```diff
@@ -4,9 +4,9 @@ import { WorkflowStep } from 'src/modules/workflow/workflow-executor/types/workf
 export interface WorkflowAction {
   execute({
     step,
-    payload,
+    context,
   }: {
     step: WorkflowStep;
-    payload?: object;
+    context?: Record<string, any>;
```

Comment:
I like to use `unknown` instead of `any` when I don't know the data type that will be there. It forces me to type-check it explicitly later.

Reply:
Also, we might want to create a specific type for the context so we can reuse it easily.

Reply:
We use `Record<string, any>` in a lot of places actually. This is because we do not know the shape of the objects. And same goes for context, since it could contain anything. Not sure what specific type we would create...

About the unknown you are right, I updated the code 👍

Reply:
Perfect for `unknown` 👌
I wasn't clear, but I meant that we could create a `WorkflowStepInputContext` type or another name not to repeat `Record<string, unknown>` many times.